### DISCUSSION TITLE
GitHub Pages[bot] build workflow has Node deprecation warning

### DISCUSSION THREAD

**Discussion by kelvinqian00** [community/community#37224] [Other Feature Feedback, Questions, & Ideas]
When deploying a new page to our github-pages branch, I noticed that the ensuing build workflow has a deprecation warning:

> Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout, actions/checkout

This warning specifically appears in the initial `build` job, at the "Checkout" step. The fix should be relatively simple - upgrade the `actions/checkout` actions from version 2 to 3. However, I was surprised to discover that not only was this not fixed, but there has been little discussion on this issue.

A screenshot from our workflow summary to give a better idea of what I am talking about:
<img width="1786" alt="Screen Shot 2022-10-25 at 11 38 12 AM" src="https://user-images.githubusercontent.com/32022576/197818789-d939737d-271c-4d38-b0e2-7a467568140d.png">

**Comment by yoannchaudet** [comment]
Thank you for reporting this. It is being tracked and should be addressed shortly.

**Reply by jhlegarreta** [reply]
This is now happening for Node.js 16, e.g.:
https://github.com/bostonmiw/bmiw/actions/runs/6642020635

Can this be addressed, please?
